public/src/js/jobtracking.js

```diff
@@ -0,0 +1,33 @@
+$(document).on('click', '.notifyjs-bootstrap-base', function() {
+    location.reload(true);
+});
+
+var taskID = setInterval(function() {
+    var div = document.getElementById('msgHolder'),
+        divChildren = div.childNodes;
+    if (divChildren.length === 0) {
+        clearInterval(taskID);
+    }
+    for (var i = 0; i < divChildren.length; i++) {
+        $.get(
+            "/api/v1/job/" + divChildren[i].getAttribute('value'), {},
+            function(data) {
```

Comment:
CC says this is a bad idea by putting a function in a loop. In reality, there's a 50% chance this code will break if there is more than one notification to present to the user (currently not possible).